Front-end/src/components/Cards.js

```diff
@@ -424,7 +424,7 @@ function Cards() {
                 </>
               );
             })
-          : filters == "aboveLand"
+          : filters === "aboveLand"
           ? aboveLand.map((element) => {
               return (
                 <>
```

Comment:
Anyways we have made some little progress for this issue because of your help so I will merge your code
Thanks for the help !!